aztk/utils.py

```diff
@@ -0,0 +1,210 @@
+import getpass
+import time
+from typing import List
+from subprocess import call
+import azure.batch.models as batch_models
+import aztk_sdk.spark
+import aztk_sdk.utils.error as error
+import aztk_sdk.utils.get_ssh_key as get_ssh_key
```

Comment:
not my biggest strength here, so please educate me if I'm wrong, but shouldn't this be
```python
from aztk_sdk.utils import error, get_ssh_key
```

Reply:
Either work -- it's mostly a style choice.